site/en/docs/privacy-sandbox/attribution-reporting/index.md

```diff
@@ -421,13 +419,10 @@ They provide conversion insights without tracking users across sites, by using t
 
 ## Engage and share feedback
 
--  For questions about the proposal:
+-  For questions about the API:
    [create an issue](https://github.com/WICG/conversion-measurement-api/issues) 
-   on the proposal repo.
--  If you're an origin trial participant and have technical questions, join the
-   [Attribution Reporting mailing list](https://groups.google.com/u/1/a/chromium.org/g/attribution-reporting-api-dev) 
-   for developers and ask questions there, or
-   [file a Chromium bug](https://bugs.chromium.org/p/chromium/issues/list?q=attribution%20reporting).
+   on the API repo.
+-  If you have technical questions, [file a Chromium bug](https://bugs.chromium.org/p/chromium/issues/list?q=attribution%20reporting).
```

Comment:
AFAIK the mailing list will continue to operate, so this could just be rephrased (as for other APIs).

Something like: "Follow API updates and announcements on the [Attribution Reporting mailing list](https://groups.google.com/u/1/a/chromium.org/g/attribution-reporting-api-dev)."

BTW: despite the earlier phrase here ("and have technical questions") these groups are really only for announcements, not discussion.